### PR TITLE
Feature/requirements installation

### PR DESCRIPTION
**Pull request by KaranBanjade**

### CODE REVIEW

File: src/meta/osMapper.js [hunk 1]
```diff
@@ -4,16 +4,37 @@ module.exports = Object.freeze({
...
         },
         {
             id: "Linux",
             os: "Linux",
             commands: { LINK: "xdg-open" },
+            installers: {
+                "Node" : `curl -fsSL https://webinstall.dev/node@stable | bash`,
+                "npm" : `npm install -g npm | bash`,
+                "ng" : `npm install -g @angular/cli | bash`,
+                "go" : `curl -fsSL https://webinstall.dev/golang@1.14 | bash`,
```
Comment (omgupta1608) [inline src/meta/osMapper.js:23]: Will only install go 1.14. This is what I was talking about.

File: src/core/meta.js [hunk 2]
```diff
@@ -26,14 +28,37 @@ module.exports = async function (meta, id) {
...
+        for (index = 0; index < requirements.length; index++) {
+        try {
+            let req = requirements[index];
+            let cmdSuccess = await execute(OS.installers[req], `Installing ${req}`);
+            if(!cmdSuccess) return cmdSuccess;
+            else {
+                print(`Installed ${req}`, "green");
+                return true;
+            }
+        } catch (err) {
```
Comment (omgupta1608) [inline src/core/meta.js:90]: log error. User needs to know what went wrong.

File: src/meta/osMapper.js (hunk 1 above)
Comment (KaranBanjade) [reply src/meta/osMapper.js:23]: Right now can be kept as static(being updated manually). Will be changed once a latest release link can be found or can be automated at further stage.
Second option could be making a script which builds the go and other things from the github repositories as that is updated regularly and link remains static

File: src/core/meta.js (hunk 2 above)
Comment (KaranBanjade) [reply src/core/meta.js:89]: changes pushed

**Comment by omgupta1608** [comment]
seems ok. merging. please deploy to npm. @KaranBanjade

**Pull request merged**